src/System.Collections.NonGeneric/ref/System.Collections.NonGeneric.cs

```diff
@@ -36,25 +35,25 @@ public abstract partial class CollectionBase : System.Collections.ICollection, S
         object System.Collections.ICollection.SyncRoot { get { throw null; } }
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
-        object System.Collections.IList.this[int index] { get { throw null; } set { } }
+        object? System.Collections.IList.this[int index] { get { throw null; } set { } }
```

Comment:
should InnerList  return nullable?

Reply:
yes, that is ArrayList and indexer return nullable